test/e2e/density.go

```diff
@@ -71,20 +70,6 @@ func printLatencies(latencies []podLatencyData, header string) {
 	Logf("perc50: %v, perc90: %v, perc99: %v", perc50, perc90, perc99)
 }
 
-// List nodes via gcloud. We don't rely on the apiserver because we really want the node ips
-// and sometimes the node controller is slow to populate them.
-func gcloudListNodes() {
```

Comment:
curious: Why is it that we no longer require this function, and now rely on apiserver?